app/javascript/components/applications/application_form/HouseholdMemberForm.js

```diff
@@ -22,13 +20,12 @@ const HouseholdMemberForm = ({ i }) => {
             <Text field={`householdMembers.${i}.lastName`} />
           </div>
           <div className="small-3 columns">
-            <label>Date of Birth <span className="checkbox-block_note no-margin">YYYY-MM-DD (required)</span></label>
-            <DatePickerText
-              required={true}
-              dateFormat="YYYY-MM-DD"
-              showYearDropdown
-              dropdownMode="select"
-              field={`householdMembers.${i}.DOB`} />
+            <label>Date of Birth</label>
+            <Text
+              field={`householdMembers.${i}.DOB`}
+              type="date"
```

Comment:
here to should we include `required={true}` ? or it is not required any more

Reply:
`required` uses the browser validation which messes with ours so we are going to reimplement it our way in https://www.pivotaltracker.com/story/show/159244905